### ISSUE TITLE
Sounds fail to load: NameError: global name 'loops' is not defined in sound.py

### ISSUE THREAD

**Issue by onesandzeroes** [psychopy/psychopy#813]
Trying to load a sound under Psychopy 1.82.00 is currently giving me `NameError: global name 'loops' is not defined`. I think the problem was introduced in [this commit](https://github.com/psychopy/psychopy/commit/0967dbf732e5e33f80102cb568c421272fd79e37). There's a few lines there where an `=` operator was used, where I'm guessing the intent was either to use a `+` or `==`? 

I'm still trying to figure out the idea behind that commit so I'm not exactly sure what the best fix would be, I'll try to submit a pull request if I come up with something.

Code to reproduce the issue:

```python
from psychopy import sound

sound = sound.Sound('Sounds/Ouch.ogg')
```

Output:

```
Traceback (most recent call last):
  File "C:/Users/me/Code/PairedUp/sound_bug_repro.py", line 3, in <module>
    sound = sound.Sound('Sounds/Ouch.ogg')
  File "C:\Program Files (x86)\PsychoPy2\lib\site-packages\psychopy-1.82.00-py2.7.egg\psychopy\sound.py", line 220, in __init__
    self.requestedLoops = self.loops = int(loops) #-1 for infinite or a number of loops
NameError: global name 'loops' is not defined
```

**Comment by peircej** [comment]
Oops. I know exactly what's gone wrong here and will fix immediately. (I tested using pyo sounds but obviously didn't check the pygame sound engine which this is affecting). Thanks for catching it
Jon

**Comment by peircej** [comment]
fixed in d88ff4a4f8ddc64ba5eaaa2599bcd21b1f740992
